subprojects/core-api/src/main/java/org/gradle/api/vcs/internal/MavenVcsMapping.java

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright 2017 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.gradle.api.vcs.internal;
+
+import org.gradle.api.vcs.VcsMapping;
+
+public interface MavenVcsMapping extends VcsMapping {
+    String getGroup();
+    String getModule();
```

Comment:
we should use `ComponentSelector` to represent identifiers in a namespace (the maven namespace in this instance).